lnrpc/verrpc/log.go

```diff
@@ -0,0 +1,32 @@
+package verrpc
+
+import (
+	"github.com/btcsuite/btclog"
+	"github.com/lightningnetwork/lnd/build"
+)
+
+// log is a logger that is initialized with no output filters.  This
+// means the package will not perform any logging by default until the caller
+// requests it.
+var log btclog.Logger
+
+// Subsystem defines the logging code for this subsystem.
+const Subsystem = "VRPC"
```

Comment:
Pretty cool that `lnd` will now ship with an integrated VR PC for the next major release 😎